### ISSUE TITLE
Avoid recreation of bastion to keep the same IDs

### ISSUE THREAD

**Issue by cjoecker** [moia-oss/bastion-host-forward#321]
Right now, I'm creating a bastion as follows:

```
new BastionHostRDSForward(scope, "Ec2Bastion", {
    vpc: vpc,
    rdsInstance: rdsInstance,
    name: "BastionHost",
  });

```

In every pipeline run, I deploy the stack. The problem is, that the bastion gets recreated on every deploy. That makes it generate a new id. 
Ideally, the bastion doesn't gets recreated and keeps the same id if it already exists.

**Comment by snowiow** [comment]
Hi, once again sorry for the late response :see_no_evil: 
Normally this shouldn't happen, because we don't set dynamic parts in any of the ids, but only pass through, whats given to the construct or set static ids. Is this still a problem in the current version? Can you share some more code around that call?

Thanks already!

**Comment by snowiow** [comment]
yes, that sounds goo. Getting the newer AMI is generally a thing we want, so I would keep that as the default but we can expose the attribute in case someone wants to set this differently :+1:  

thanks for pointing this out @NikitaKemarskiy :pray:

**Comment by snowiow** [comment]
It's available with v2.1.0
